opensoundscape/audio.py

```diff
@@ -266,79 +266,69 @@ def split(self, clip_duration, clip_overlap=0, final_clip=None):
         The Audio object is split into clips of a specified duration and overlap
 
         Args:
-            clip_duration:  The duration in seconds of the clips
-            clip_overlap:   The overlap of the clips in seconds [default: 0]
-            final_clip:     Possible options (any other input will ignore the final clip entirely),
-                                - "remainder":          Include the remainder of the Audio
-                                                            (clip will not have clip_duration length)
-                                - "full":               Increase the overlap to yield a clip with clip_duration
-                                - "extend":             Similar to remainder but extend the clip to clip_duration
+            clip_duration (float):  The duration in seconds of the clips
+            clip_overlap (float):   The overlap of the clips in seconds [default: 0]
+            final_clip (str):       Behavior if final_clip is less than clip_duration seconds long. [default: None]
+                By default, ignores final clip entirely.
+                Possible options (any other input will ignore the final clip entirely),
+                    - "remainder":  Include the remainder of the Audio (clip will not have clip_duration length)
+                    - "full":       Increase the overlap to yield a clip with clip_duration length
+                    - "extend":     Similar to remainder but extend (repeat) the clip to reach clip_duration length
```

Comment:
I thought the extend behavior would extend the clip with silence - didn't it used to behave like that? 
Extend function is misleading - Maybe we should make Audio.extend() add silence and call the current function audio.repeat() or something.

Reply:
To the best of my knowledge, we always used "extend" to mean "loop the recording so it extends to the desired length" -- I think this was modeled after a Lasseck approach

+1 to the idea of changing `Audio.extend()` to either `Audio.repeat()`  or `Audio.loop()` then adding an "extend with silence" method

Reply:
Added an enhancement request for this (#265) but will not change in this PR